.github/workflows/lint.yml

```diff
@@ -9,15 +9,15 @@ on:
       - main
 
 jobs:
-  # https://github.com/avto-dev/markdown-lint
+  # https://github.com/nosborn/github-action-markdown-cli
```

Comment:
Will continue to use this action, 10-20 seconds faster than the one I was using before